hardhat.config.js

```diff
@@ -22,22 +22,31 @@ const getBooleanValue = (input) => {
     : false;
 };
 
+const getArrayValue = (input) => {
```

Comment:
`const getBooleanValue` have got unused. You have to get rid of it.

The same for `axios`